kroxylicious-bom/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Missing license, why isn't that caught be the build?

Reply:
Very good question it fails every other new thing I touch...

Reply:
028e50b307f2d552bd5619971c3d27c7bea89c0e

Reply:
So the header itself was fixed in 028e50b307f2d552bd5619971c3d27c7bea89c0e and I added the plug-in to check in 81a5fe6fc68ef79dcda72466707e9758b86f0033. 

The TL;DR summary is the bom doesn't use `kroxylicious-parent` (to avoid circular dependencies) so didn't inherit the plug-in settings to validate the header.